### PR TITLE
[Merge After V2 release only!] Upgrade github documentation to v2.0.0 on master branch

### PR DESCRIPTION
**Pull request by gargnitingoogle**
### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

**Review by ashmeenkaur — CHANGES_REQUESTED** [review]
Can you please pick up the changes from latest read cache release. I don't see min object changes in the semantics doc.

**Comment by gargnitingoogle** [comment]
> Can you please pick up the changes from latest read cache release. I don't see min object changes in the semantics doc.

Good catch. I'll bring that in as well. Thanks @ashmeenkaur .

File: docs/semantics.md [hunk 1]
```diff
@@ -62,22 +62,42 @@ They are discussed in this section, along with their trade-offs and the situatio
 
 The default behavior is appropriate, and brings significant performance benefits, when the bucket is never modified or is modified only via a single Cloud Storage FUSE mount. If you are using Cloud Storage FUSE in a situation where multiple actors will be modifying a bucket, be sure to read the rest of this section carefully and consider disabling caching.
 
-Important: The rest of this document assumes that caching is disabled (by setting ```--stat-cache-ttl 0``` and ```--type-cache-ttl 0```). This is not the default. If you want the consistency guarantees discussed in this document, you must use these options to disable caching.
+**Important**: The rest of this document assumes that caching is disabled (by setting ```--stat-cache-ttl 0``` and ```--type-cache-ttl 0```). This is not the default. If you want the consistency guarantees discussed in this document, you must use these options to disable caching.
```
Comment (sethiay) [inline docs/semantics.md:65]: can we say by setting ```--stat-cache-ttl 0``` and ```--type-cache-ttl 0``` or ```metadata-cache:ttl-secs: 0``` ?

Comment (gargnitingoogle) [reply docs/semantics.md]: done

**Review by sethiay — APPROVED** [review]

**Pull request merged**